src/schemas/public/MeasurementType.ts

```diff
@@ -6,14 +6,12 @@ import { z } from 'zod';
 /** Represents the enum public.measurement_type */
 type MeasurementType = 
   | 'boolean'
-  | 'dataframe'
-  | 'scalar';
+  | 'dataframe';
 
 export default MeasurementType;
 
 /** Zod schema for measurement_type */
 export const measurementType = z.enum([
   'boolean',
   'dataframe',
-  'scalar',
 ]);;
```

Comment:
maybe you need to rerun db migrations? migrate all the way down then migrate to latest again

Reply:
@smahmed776